src/libsyntax/feature_gate/active.rs

```diff
@@ -533,6 +530,12 @@ declare_features! (
     /// Allows using the `efiapi` ABI.
     (active, abi_efiapi, "1.40.0", Some(65815), None),
 
+    /// Allows using the `#[register_attr]` attribute.
+    (active, register_attr, "1.41.0", Some(29642), None),
+
+    /// Allows using the `#[register_attr]` attribute.
+    (active, register_tool, "1.41.0", Some(44690), None),
```

Comment:
I would prefer new fresh tracking issues for both of these to have more targeted discussion and triage.

Reply:
Tracking issue for tools - https://github.com/rust-lang/rust/issues/66079.

Reply:
Tracking issue for attrs - https://github.com/rust-lang/rust/issues/66080.